### ISSUE TITLE
Add gender data to populator or to data files / better gender handling

### ISSUE THREAD

**Issue by mathfreak231** [TwitchPlaysPokemon/pokecat#8]
Pokecat should have some kind of gender checking to make sure Pokemon are given the correct gender. To prevent PBR crashing all it needs is a gendered vs genderless check, but checking for male-only or female-only Pokemon would be helpful as well. Currently gender is not checked at all by pokecat itself to my knowledge.

A better default for gender would also be useful: default to `null`, `m`, `f`, or `[m, f]` depending on the gender of that species.

I believe this fits under the "error tolerance against common mistakes" requirement.

**Comment by Felk** [comment]
I'm all for a better default, but the PBR-crash-check in question is difficult as pokecat operates on a per-pokeset basis, so the individual processes don't have any shared context. However, exactly that check is already implemted in [pokeset-ci](https://github.com/TwitchPlaysPokemon/pokeset-ci/blob/master/pokesetci/analyze.py#L146), but that was shut down somewhen (or stopped working and noone cared, I don't exactly know).

**Comment by mathfreak231** [comment]
Yes, checking against each species' gendering

also wow I didn't know we had CI for pokesets! I'll have to check it out and maybe help rewrite it if it's needed

**Comment by Felk** [comment]
I don't think it needs a rewrite, it works fine. Given that [this pokeset-ci issue](https://github.com/TwitchPlaysPokemon/pokeset-ci/issues/2) is resolved and the data adjusted accordingly, I could just start it again if I get a github token with the webhook creation permission (see the [config_default.yml](https://github.com/TwitchPlaysPokemon/pokeset-ci/blob/master/config_default.yml)
